concordium-std-derive/src/lib.rs

```diff
@@ -376,7 +382,20 @@ fn receive_worker(attr: TokenStream, item: TokenStream) -> syn::Result<TokenStre
 
     let fn_name = &ast.sig.ident;
     let rust_export_fn_name = format_ident!("export_{}", fn_name);
-    let wasm_export_fn_name = format!("{}.{}", contract_name, name);
+    let wasm_export_fn_name = format!("{}.{}", contract_name.value(), name.value());
```

Comment:
Is there ever a case where you don't use `.value()` on the result of `get_attribute_...`?

If so, I think it is a bad idea to change the return type as you did.

Reply:
No, there isn't. But I made the change so that we could use `span()` on the `LitStr` and thereby show the correct position of the error.